src/NetAnalyzers/UnitTests/Microsoft.NetCore.Analyzers/Security/DataSetDataTableInSerializableTypeTests.cs

```diff
@@ -399,8 +510,18 @@ private static DiagnosticResult GetNonIFormatterCSharpResultAt(int line, int col
                 .WithArguments(arguments);
 
         private static DiagnosticResult GetIFormatterCSharpResultAt(int line, int column, params string[] arguments)
-    => VerifyCS.Diagnostic(DataSetDataTableInSerializableTypeAnalyzer.RceSerializableContainsDangerousType)
-        .WithLocation(line, column)
-        .WithArguments(arguments);
+            => VerifyCS.Diagnostic(DataSetDataTableInSerializableTypeAnalyzer.RceSerializableContainsDangerousType)
+                .WithLocation(line, column)
+                .WithArguments(arguments);
+
+        private static DiagnosticResult GetAutogeneratedNonIFormatterCSharpResultAt(int line, int column, params string[] arguments)
```

Comment:
[minor] Having the exact parameters makes it easier for external contributors to understand what's expected (exact number + exact content).